### ISSUE TITLE
Switch to CSC colour scheme and styling

### ISSUE THREAD

**Issue by lilachic** [CSCfi/metadata-submitter-frontend#363]
#### Description

CSC UX group provides CSC design system, we would like to to switch our UX according to that.

CSC styling system should bring along CSC colours, SCS styling  and accessibility.

for now we are not going to use it due to missing components:
> CSC design system:
> https://cscui.a3s.fi/index.html 

Tips about accessibility:
https://webaim.org/articles/contrast/
https://wave.webaim.org/extension/

Relates to #170 and #105 as other design issues.

#### DoD (Definition of Done)
When the UX colours and styling mainly conforms CSC design  system, accessibility checked.

#### Testing
Peer review.

**Comment by lilachic** [comment]
Help icons, `helpIconStyle` , question marks at forms need to be improved in terms of accessibility, contrast.

**Comment by blankdots** [comment]
the lack of UI components and accessibility and we cannot ahead and use the CSC design system.

**Comment by hannyle** [comment]
Here I got the latest color guidelines for the UI.
- Background color: Darkgreen 10%

[CSC Design system + instructions.pdf](https://github.com/CSCfi/metadata-submitter-frontend/files/7525385/CSC.Design.system.%2B.instructions.pdf)
